static/js/settings_account.js

```diff
@@ -329,7 +329,7 @@ exports.set_up = function () {
         channel.patch({
             url: "/json/settings",
             data: data,
-            beforeSubmit: function () {
+            beforeSend: function () {
                 if (page_params.realm_password_auth_enabled !== false) {
                     // FIXME: Check that the two password fields match
                     // FIXME: Use the same jQuery validation plugin as the signup form?
```

Comment:
Hmm, we might want to do some follow-up work with these FIXMEs; I'm not sure I trust that code :)

Reply:
BTW, I think this was a feature of `jquery-form`, a plugin we removed in: 07b63e4886664f0db4c88b2002ecb403ab51be28.  So the error was introduced in b94986b1e1061ee7349ba9eb06228b6b6b7f8623.